modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console to see how the traffic moves around the application, in the menu bar, click the *Application launcher* > *Kiali*.
```

Comment:
I think there is another spot where it can be accessed like the project launcher.
cc @cfcosta

Reply:
Yes, on the project details page there’s also a link to Kiali.

Reply:
That's true, but I left it out on purpose. I think that the user experience we want to highlight is the most simple one, which is finding Kiali in the application launcher.